src/overlays/actors/ovl_En_Encount3/z_en_encount3.c

```diff
@@ -32,22 +33,174 @@ ActorInit En_Encount3_InitVars = {
     (ActorFunc)EnEncount3_Draw,
 };
 
-#endif
+u32 D_809AD810 = false;
```

Comment:
Does it need to be unsigned?
```suggestion
s32 D_809AD810 = false;
```